pkg/ebpf/tcp_queue_length_types.go

```diff
@@ -0,0 +1,29 @@
+package ebpf
+
+import (
+	"net"
+)
+
+// QueueLength contains the size and fullness extremums of a TCP Queue
+type QueueLength struct {
+	Size int    `json:"size"`
+	Min  uint32 `json:"min"`
+	Max  uint32 `json:"max"`
+}
+
+// Conn contains a TCP connection quadruplet
+type Conn struct {
```

Comment:
Can we perhaps move these types to its own package?
I think it's definitely fair to leave `TCPQueueLengthTracer` in the eBPF package, but `Conn` and `Stats` are overloaded terms that will clash with other stuff.

Reply:
Good point !
I’ve just moved those data structures in a dedicated package.